masterlist.yaml

```diff
@@ -9612,7 +9612,7 @@ plugins:
     after:
       - 'Requiem - Legendary Bugsmasher Edition.esp'
       - 'Requiem_Minor_Arcana.esp'
-  - name: 'Sacrosanct Requiem Patch and Rebalance.esp'
+  - name: 'Requiem - Sacrosanct Patch.esp'
```

Comment:
You'll need to keep the old version name as well.
Just in case they're using the old version.

```yaml
  - name: '(Sacrosanct Requiem Patch and Rebalance|Requiem - Sacrosanct Patch).esp'
```